tests/rule_based_profiler/conftest.py

```diff
@@ -2404,3 +2417,73 @@ def rule_with_variables_with_parameters(
         column_Date_domain.id: multi_part_name_parameter_container,
     }
     return rule
+
+
+# Fixtures from config_version 1.0 ============================================================================
+
+
+@pytest.fixture(scope="function")
+def profiler_name() -> str:
+    return "my_first_profiler"
```

Comment:
These fixtures have wider applicability than their origin file so moving to conftest to reuse.